src/lib/layer.js

```diff
@@ -62,7 +62,12 @@ const defaultProps = {
   // Offset depth based on layer index to avoid z-fighting.
   // Negative values pull layer towards the camera
   // https://www.opengl.org/archives/resources/faq/technical/polygonoffset.htm
-  getPolygonOffset: ({layerIndex}) => [0, -layerIndex * 100]
+  getPolygonOffset: ({layerIndex}) => [0, -layerIndex * 100],
+
+  // Selection/Highlighting
+  selectedObjectIndex: -1,
```

Comment:
Note for API review: I am wondering if this should be `highlightedObjectIndex` for consistency. `selected` could mean various things...